client/components/groupevents/modal.jsx

```diff
@@ -9,133 +9,134 @@ import React from 'react';
  */
 export default class Modal extends React.Component {
```

Comment:
Be consistent in naming component files. If you want to use capital first letters (preferable) all through, stick with it.